api/src/main/java/org/cardanofoundation/tokenmetadata/registry/api/controller/HealthApiController.java

```diff
@@ -0,0 +1,31 @@
+package org.cardanofoundation.tokenmetadata.registry.api.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.cardanofoundation.tokenmetadata.registry.api.model.rest.HealthResponse;
+import org.cardanofoundation.tokenmetadata.registry.model.enums.SyncStatusEnum;
+import org.cardanofoundation.tokenmetadata.registry.service.TokenMetadataSyncService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@CrossOrigin
+@RequestMapping("${openapi.metadataServer.base-path:}")
+@Slf4j
+@RequiredArgsConstructor
+public class HealthApiController implements HealthApi {
+
+    private final TokenMetadataSyncService tokenMetadataSyncService;
+
+    @Override
+    public ResponseEntity<HealthResponse> getHealthStatus() {
+
+        return new ResponseEntity<>(HealthResponse.builder()
+                .synced(tokenMetadataSyncService.isInitialSyncDone())
```

Comment:
need to be carefull here, because if the api is starting w/o the Scheduled Job enabled, the `tokenMetadataSyncService` will never be called and hence always return false.

You should probably check for the scheduledJob flag to be enabled
1.  and then only then check the sync status
2. return true otherwise

Reply:
Sure will do it like this!

Reply:
I will add true and for the enum (the status text). I'll return that the syncing is done by the external job.